frameworks/helloworld/src/main/dist/examples/marathon_constraint.yml

```diff
@@ -0,0 +1,43 @@
+name: "hello-world"
+pods:
+  hello:
+    count: {{HELLO_COUNT}}
+    placement: {{HELLO_PLACEMENT}}
```

Comment:
Could this use the default svc.yml for hello-world? Looks like that has placement constraint fields defined